src/io-methods/google-cloud-storage.js

```diff
@@ -18,15 +20,16 @@ function parseFilePath(filePath) {
   return { bucketName, fileName };
 }
 
-exports.readManifest = function (filePath) {
-  return Promise.resolve().then(() => {
-    const { bucketName, fileName } = parseFilePath(filePath);
-    return storage
-      .bucket(bucketName)
-      .file(fileName)
-      .download()
-      .then((data) => data.toString("utf-8"));
-  });
+exports.readManifest = async function (filePath) {
+  const { bucketName, fileName } = parseFilePath(filePath);
+  const file = await storage.bucket(bucketName).file(fileName);
+  const exists = first(await file.exists());
```

Comment:
Pretty weird that file.exists() returns `[true]` or `[false]` - i was initially curious why `first()` was needed but then found the following links and it makes sense:

https://googleapis.dev/nodejs/storage/latest/global.html#FileExistsResponse

https://googleapis.dev/nodejs/storage/latest/File.html#exists